rllib/agents/trainer.py

```diff
@@ -81,7 +81,9 @@
     # Number of GPUs to allocate to the trainer process. Note that not all
     # algorithms can take advantage of trainer GPUs. This can be fractional
     # (e.g., 0.3 GPUs).
-    "num_gpus": 0,
+    # None (default): Use all available GPUs.
+    # 0: Will not use any GPUs, even if one is available.
+    "num_gpus": None,
```

Comment:
New default for num_gpus: None -> Figure out GPU placement based on what's available.
Also: num_gpus=0 means: Do NOT use any GPUs.